### ISSUE TITLE
not possible to call initializeDateFormatting('pt_BR',null) as described in pub.dev

### ISSUE THREAD

**Issue by insinfo** [dart-lang/i18n#856]
it is not possible to call initializeDateFormatting('pt_BR',null) as described in pub.dev because in intl-0.19.0 it is requiring a second filePath argument (which I don't know what it is)

![image](https://github.com/dart-lang/i18n/assets/12227024/53fea231-5880-43c8-b304-659abd7df03a)

![image](https://github.com/dart-lang/i18n/assets/12227024/2acdd31d-be42-4217-be55-251998ada681)
Dart SDK version: 3.2.1 (stable) (Wed Nov 22 08:59:13 2023 +0000) on "windows_x64"

**Comment by Igor0155** [comment]
Certifica que o import seja do local:

import 'package:intl/date_symbol_data_local.dart';

**Comment by insinfo** [comment]
I ended up discovering this on my own, but ideally this would be better documented in the intl package.

**Comment by mosuem** [comment]
@insinfo Yeah, the different `initializeDateFormatting` are pretty confusing. Any idea where this should best be documented? I would be happy to add some help around this frequent issue, and you might know best given that you recently ran into this.
